File: src/main/java/econtroller/controller/Sense.java
package econtroller.controller;

import java.util.List;

import se.sics.kompics.Event;
import se.sics.kompics.address.Address;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-04-11
 *
 */

public class Sense extends Event {

	private Address cloudProvider;

	public Sense(Address cloudProviderAddress) {
		this.cloudProvider = cloudProviderAddress;
	}

	public Address getCloudProvider() {
		return cloudProvider;
	}
	
}

File: src/main/java/econtroller/controller/Sense.scala
package econtroller.controller

import se.sics.kompics.Event
import se.sics.kompics.address.Address

/**
 * @author Amir Moulavi
 * @date 2011-07-23
 *
 */
class Sense(val cloudProvider:Address) extends Event {
  def getCloudProvider = cloudProvider
}